src/wp-includes/class-wp-term-query.php

```diff
@@ -160,6 +160,7 @@ class WP_Term_Query {
 	 *                                                   (even if `$hide_empty` is set to true). Default true.
 	 *     @type string          $search                 Search criteria to match terms. Will be SQL-formatted with
 	 *                                                   wildcards before and after. Default empty.
+	 *     @type string          $s                      Alias of `$search`. Default empty.
```

Comment:
Add `@since` for new variable.

Reply:
Hey @mukeshpanchal27 , since we have version number like this in the core `wp_version = '6.6-beta1-58337-src';` . What WP Version should I write for the `@since`.

Reply:
Since it is an enhancement and 6.6 is in beta, then this would probably need to be 6.7.

Reply:
Agree with @westonruter